### ISSUE TITLE
Adding Hyperlink breaks Word document 

### ISSUE THREAD

**Issue by dannycornelisseMT** [dolanmiu/docx#390]
I'm on 5.0.0rc5. When I try to add a hyperlink to the document, I get the following message (while opening the word document): 'Word found unreadable content in "example (5).docx". Do you want to recover the contents of this document? If you trust the source of this document, click Yes.'

See the stackblitz below, note the versions of docx and file-saver in package.json

https://stackblitz.com/edit/angular-cny1zr?file=src/app/app.component.ts

Am I adding the hyperlinks wrong? 

Thanks in advance!

P.s. please excuse my not using a proper bug report template.

Edit, I'm on MacOS by the way

**Comment by dannycornelisseMT** [comment]
@dolanmiu Is this something that is known and/or being worked on? Otherwise, I'll move on to an alternative solution.

p.s. Great library, awesome work

**Comment by omgovich** [comment]
Hi, I have the same problem:

After adding a link to the document, MS Word can't open it without alert (tested it on Windows and macOS):
![photo5195323579934288889](https://user-images.githubusercontent.com/206567/65312641-9d7adf00-db9b-11e9-8af8-829d80db2e47.jpg)
 
I used the code from this example:
https://github.com/dolanmiu/docx/blob/master/demo/35-hyperlinks.ts

Package version: `5.0.0rc5`

Resulting document example: 
https://drive.google.com/file/d/1HlGzCMdbTVUZw2cP24hXgB9BycGkRG0T/

**Comment by Sraleik** [comment]
same problem on version: `5.0.0-rc7`

```javascript
new Paragraph({
  children: [
    document.createHyperlink('https://www.google.com', 'Awesome link')
  ]
})
```

Is this the right way to create a link ?

**Comment by Sraleik** [comment]
I found the problem 

in document.xml.rels my hyperlink as a duplicate id (here rId5) :

![duplicate_id](https://user-images.githubusercontent.com/9767889/68408949-92e5ca80-0186-11ea-9268-a2f77f4aefea.jpg)

i dont know how if it's the hyperlink that as the wrong relationshipsCount or if it's header || footer

but removind those duplicate ids fixes my problem

**Comment by Sraleik** [comment]
#440 fix it for me too

**Comment by dannycornelisseMT** [comment]
@Sraleik How did #440 fix it for you?

**Comment by Sraleik** [comment]
as said by @ramonmata
It's a hacky fix. 
I'm trying to find a better one

but for now modifying the library like that works :

https://github.com/ramonmata/docx/blob/104c4c4deb84e1a5deb47ac7fc4dc156632f4cc6/src/file/relationships/relationships.ts#L27

**Comment by omgovich** [comment]
@dolanmiu We could just add some method like lodash's [`uniqueId`](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L16170) which will return uniq numbers.

What do you think about this approach?

**Comment by dolanmiu** [comment]
I think this PR does just that, it uses a well known library to generate the unique ID. Only problem is that it includes letters and numbers, but apparently it still works as expected:

https://github.com/dolanmiu/docx/pull/448

I will test it out locally before I merge

**Comment by dannycornelisseMT** [comment]
Thanks @dolanmiu! You're a lifesaver, really wanted this issue to be fixed

**Comment by dolanmiu** [comment]
Fixed with #448
